x-pack/plugins/uptime/public/components/queries/monitor_status_bar/monitor_status_bar.tsx

```diff
@@ -99,19 +98,11 @@ export const MonitorStatusBar = ({
               />
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <FormattedMessage
-                id="xpack.uptime.monitorStatusBar.healthStatus.hostMessage"
-                values={{ host }}
-                defaultMessage="Host: {host}"
-              />
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <FormattedMessage
-                id="xpack.uptime.monitorStatusBar.healthStatus.portMessage"
-                values={{ port }}
-                defaultMessage="Port: {port}"
-              />
```

Comment:
Ah, so I removed this because the formatting got weird and split across lines doubling the height of the bar.

I'm thinking maybe none of these things need labels? But maybe we can do that in a follow-up PR? For the time being it's pretty obvious that the URL is a URL without a label being there.

Reply:
👍